[BLOG] Chocolatey Update

Daeraxa opened the pull request:
Small blog post to update the community on the recent work done to make sure we can support Chocolatey again.

DeeDeeG commented on docs/blog/20230825-Daeraxa-ChocolateyUpdate.md:
```diff
@@ -0,0 +1,29 @@
+
+As you may know, we decided to officially support the [Chocolatey](https://chocolatey.org/) package manager on Windows back in April. However, we quickly ran into a few problems regarding the Pulsar binaries exceeding the size allowed. Unfortunately, this meant that Pulsar was "stuck" on version 1.103.0 whilst we tried to find a solution.
+
+Unfortunately, this went on for rather longer than expected, but thanks to the efforts of a number of people, including @confused-techie, @spiker985 and @COLAMAroro, a solution was found to the problem, and, as a result, the latest regular release of Pulsar is once again available on Chocolatey. Not only that, but all the intermediate releases have also been provided.
```
Consider adding links to their github accounts, if already `@mention`ing them, but whether to do so is a subjective choice, so I leave it to your judgment as the post author.

Daeraxa replied:
Thats a good point, I missed that out, usually I have a common section at the bottom of my community posts to do links like that. I'll add those in

DeeDeeG reviewed (approved):
I can't speak to reviewing accuracy, I'm honestly catching up with how this went by reading the post, but I don't know of anything _wrong_ in it, seems (in a general sense) to match what I recall @confused-Techie saying about it elsewhere.

Otherwise, grammar/spelling all looks good, and it ran fine for me locally, so no syntax errors or technical issues that I'm aware of from a quick test.

LGTM, then!

(I know I don't usually review website things, but I was available and I thought I'd give it a shot!)

DeeDeeG commented:
Posting a screenshot, because why not:

<details><summary>Screenshot (click to expand):</summary>

![The "Chocolatey packages are up to date again!" blog post, as run in a development copy of the site on my local machine](https://github.com/pulsar-edit/pulsar-edit.github.io/assets/20157115/b701160c-05ad-418a-9b6a-0dcffab09b74)

</details>

Daeraxa commented:
Couple of changes:
- Added GH links to people
- Fixed image sizes to display side by side (at least for standard resolutions and website page width)

confused-Techie reviewed (approved):
Looks super rad!

Everything here seems super accurate, and factually correct.

We can either merge this as is, or if I can be a bit extra, one change I'd like to see is if we actually moved `COLAMAroro`s name to the first position of that list.

Since while Spiker and I were the ones to work out the automatic publishing of new versions, and I was the one to get all the new ones published, they were the sole creator of the fix that allowed us to publish all the new versions. So if we felt it made sense, that does provide them a bit more importance to the list as I feel like they deserve.

But again, we can still hit merge on this one without that change

confused-Techie commented on docs/blog/20230825-Daeraxa-ChocolateyUpdate.md:
```diff
@@ -0,0 +1,33 @@
+
+As you may know, we decided to officially support the [Chocolatey](https://chocolatey.org/) package manager on Windows back in April. However, we quickly ran into a few problems regarding the Pulsar binaries exceeding the size allowed. Unfortunately, this meant that Pulsar was "stuck" on version 1.103.0 whilst we tried to find a solution.
+
+Unfortunately, this went on for rather longer than expected, but thanks to the efforts of a number of people, including [@confused-techie], [@spiker985] and [@COLAMAroro], a solution was found to the problem, and, as a result, the latest regular release of Pulsar is once again available on Chocolatey. Not only that, but all the intermediate releases have also been provided.
```
```suggestion
Unfortunately, this went on for rather longer than expected, but thanks to the efforts of a number of people, including [@COLAMAroro], [@spiker985] and [@confused-techie], a solution was found to the problem, and, as a result, the latest regular release of Pulsar is once again available on Chocolatey. Not only that, but all the intermediate releases have also been provided.
```

Daeraxa replied:
No I'll take that, you collectively did the work so you get to pick the order, I think that is more than fair.

confused-Techie reviewed (approved):
LGTM! Thanks for taking my suggested change, lets get this one merged!
